stage2/04-install-umbrel/01-run.sh

```diff
@@ -22,17 +22,31 @@ EOF
 # Install Umbrel
 echo "Installing Umbrel..."
 echo
+if [ -z ${UMBREL_REPO+x} ]; then
```

Comment:
Whats the ```+x``` part do?

Reply:
I will remove it here, I normally do this since I saw it on StackOverflow: https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash. It checks if the variable is set, even if it's empty. This is bad here, I just did it automatically.

Reply:
Oh I see .. thanks